cli/search/search.go

```diff
@@ -37,11 +40,28 @@ func SearchVersions(program string) error {
 	} else if program == "tt" {
 		cmd = exec.Command("git", "-c", "versionsort.suffix=-", "ls-remote", "--tags", "--sort="+
 			"v:refname", "https://github.com/tarantool/tt.git")
+	} else if program == "tarantool-ee" {
```

Comment:
I think that it would be better to move Tarantool-ee search into different function and call it here.

Reply:
What for? it called just after
```go
log.Warn("Available versions of " + program + ":")
```
and returns. So we don't need to duplicate code.